tools/build/Makefile.rules

```diff
@@ -132,6 +132,23 @@ parse-bin = \
 	$(eval $(1)-deps     := $(subst .mod,,$(bin-$(1)-y-deps))) \
 	$(eval bins-out      += $(bin-$(1)-out)) \
 
+define install-resource
+$(2): $(1)
+	$(Q)echo "    "INST"   "$(1)
+	$(Q)$(MKDIR) -p $(dir $(2))
+	$(Q)$(CP) $(1) $(2)
+endef
```

Comment:
it was used manually at Makefile.target, each resource was installed with a rule there, see GDB_ variables in Makefile.vars and Makefile.targets